graphql/resolvers.go

```diff
@@ -1943,6 +1943,12 @@ func (r *mutationResolver) ScheduleUndispatchedBaseTasks(ctx context.Context, pa
 		}
 		scheduledTasks = append(scheduledTasks, task)
 	}
+
+	// Sort ensures that tasks are returned in a consistent order
+	sort.Slice(scheduledTasks, func(i, j int) bool {
```

Comment:
Are we doing this just for the test? Since this is just sending task information to the backend, it seems unnecessary to add complexity. Is there something we can do in the test instead?

Reply:
There isn't anything we can do for the test unfortunately since the test suite just compares the result of the query to an expected result. This test flakes in an unpredictable way so this seemed like the simplest solution to ensure there is always a consistent order in the response.

Reply:
I'd still rather not do this; we're adding an O(nlogn) function just for testing but it's still being applied to every use of this function. Can we pass something into FindTasksByVersion so that we sort as part of the aggregation instead? Otherwise a lot of this logic can be moved into a task function and tested with a unit test instead.